<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SorFilterDialog</class>
 <widget class="QWidget" name="SorFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistical Outlier Removal</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(221, 221, 221);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">background-color: transparent;</string>
          </property>
          <property name="text">
           <string>Number of evaluation points (â‰¥2)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="knnSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>360</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="suffix">
           <string>pts</string>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">background-color: transparent;</string>
          </property>
          <property name="text">
           <string>Deletion level</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="nSigmaDoubleSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>360</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Save path</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="minimumSize">
             <size>
              <width>272</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>272</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{padding-left:12px;border-radius: 2px;border: 1px solid #989898;color:#F2F2F2;}
QLineEdit:disabled{color:#838383;border: 1px solid #666666;background-color: #3F3F3F}
</string>
            </property>
            <property name="placeholderText">
             <string>Select a save path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{border: 1px solid #555555;
			border-radius: 2px; 
			background: #555555; 
			color: #F2F2F2; }
QPushButton:hover { background-color: #2B2B2B; }
QPushButton:disabled { color: #838383;background-color:#555555;}</string>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
